test/loading.jl

```diff
@@ -1,4 +1,16 @@
 
+@testset "tailwind" begin
```

Comment:
Is this a copy paste error? It seems to check twice the same here, but the second still has the xkcd name from the next test?

Reply:
Yes, it definitely was. 😅